.github/workflows/universal-selfhosted-backup.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         lfs: ${{ inputs.use_git_lfs }}
         fetch-depth: 0 # Make deep copy
     - name: Setup SSH
-      uses: kielabokkie/ssh-key-and-known-hosts-action@v1
+      uses: kielabokkie/ssh-key-and-known-hosts-action@v1.4.0
```

Comment:
@okalman Ondro tady by to mělo fungovat imho tak, že ta v1 stahuje vždy nejnovější release 🤔. Vycházel jsem odsud: https://devopsjournal.io/blog/2022/10/19/How-GitHub-Actions-versioning-works. Mělo by to vždy checkoutnout commit s tagem v1 a dle zvyklostí by to měli tvůrci těch actions vždy posouvat na ten aktuální. Když se dívám do toho repa https://github.com/kielabokkie/ssh-key-and-known-hosts-action/tags tak vidím že ten poslední commit má tagy v1.4.0 tag a právě i v1. Tak jen dotaz jestli s tím byl nějaký problém nebo jestli jsem nepochopil jak to funguje. Nebo bereš že je více safe mít natvrdo tu verzi? Díkec
<img width="427" alt="Screenshot 2024-02-10 at 8 22 46" src="https://github.com/futuredapp/.github/assets/8895732/3c045eff-cc1f-4151-9ed5-1b0f66170cb7">

Reply:
Ha máš pravdu a vypadá to (podle času kdy udělal ten tag, že je to právě ten poslední release co to rozbil.... failuje nějakej poslední post setup krok: https://github.com/futuredapp/salt-android/actions/runs/7845329221

udělám úpravu aby jsme používali tu starší verzi